Resources/Prototypes/Entities/Mobs/NPCs/simplemob.yml

```diff
@@ -79,7 +79,7 @@
         Heat: 1 #per second, scales with number of fire 'stacks'
   - type: Temperature
     heatDamageThreshold: 360
-    coldDamageThreshold: 260
+    coldDamageThreshold: -150
     currentTemperature: 310.15
```

Comment:
Isn't temperature in kelvin?

Reply:
Yes, but it's well below 0K just in case.

Reply:
just make it 0k